python_toolbox/evaluation/plot.py

```diff
@@ -36,53 +36,80 @@
 import matplotlib.pyplot as plt
 from cycler import cycler
 
-def plot_graph(scene, fscore, dist_threshold, edges_source, cum_source,
-		edges_target, cum_target, plot_stretch, mvs_outpath,
-		show_figure = False):
-	f = plt.figure()
-	plt_size = [14, 7]
-	pfontsize = 'medium'
 
-	ax = plt.subplot(111)
-	label_str = 'precision'
-	ax.plot(edges_source[1::], cum_source*100, c='red',
-			label=label_str, linewidth=2.0)
+def plot_graph(
+    scene,
+    fscore,
+    dist_threshold,
+    edges_source,
+    cum_source,
+    edges_target,
+    cum_target,
+    plot_stretch,
+    mvs_outpath,
+    show_figure=False,
+):
```

Comment:
could you add requirements.txt file in the root, so the matplotlib friendly functions should produce the same results with your testing environment?

Reply:
I added a requirements.txt.